source/code/troubleshooter/omsagent_tsg_zip/tsg_files/tsg_code/custom_logs/tsg_checkclconf.py

```diff
@@ -0,0 +1,106 @@
+import os
+import subprocess
+
+from tsg_errors import tsg_error_info, get_input
+
+clconf_path = "/etc/opt/microsoft/omsagent/conf/omsagent.d/customlog.conf"
+omsconflog_path = "/var/opt/microsoft/omsconfig/omsconfig.log"
+omsconflogdet_path = "/var/opt/microsoft/omsconfig/omsconfigdetailed.log"
+
+
+
+def no_clconf():
+    # check if enough time has passed for agent to pull config from OMS backend
+    print("--------------------------------------------------------------------------------")
+    print(" The troubleshooter cannot find the customlog.conf file. If the custom log \n"\
+          " configuration was just applied in portal, it takes up to 15 minutes for the \n"\
+          " agent to pick the new configuration.\n\n")
+    too_long = get_input("Has it been at least 15 minutes since applying the configuration to the\n OMS backend? (y/n)",\
+                            (lambda x : x in ['y','yes','n','no']),\
+                            "Please type either 'y'/'yes' or 'n'/'no' to proceed.")
+
+    # >=15 minutes since last applying config
+    if (too_long.lower() in ['y','yes']):
+        # TODO: check the log files for an error in DSC
+        tsg_error_info.append((omsconflog_path, omsconflogdet_path))
+        return 153
+
+    # <15 minutes since last applying config
+    else:
+        print(" Please try waiting for the agent to pull the config from the OMS backend.")
```

Comment:
suggestion, include the command they can use to manually pull 
`sudo su omsagent -c 'python /opt/microsoft/omsconfig/Scripts/PerformRequiredConfigurationChecks.py'`

Reply:
Updated the print statement to include the command:
https://github.com/microsoft/OMS-Agent-for-Linux/blob/bff7c80f869d6f5a4046ed41b4bd0ed89d69219f/source/code/troubleshooter/modules/custom_logs/check_clconf.py#L16-L20